src/Symfony/Component/HttpKernel/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ CHANGELOG
  * added `Symfony\Component\HttpKernel\Controller\ArgumentResolverInterface`
  * added `Symfony\Component\HttpKernel\Controller\ArgumentResolverInterface` as argument to `HttpKernel`
  * added `Symfony\Component\HttpKernel\Controller\ArgumentResolver`
+ * added the `kernel.controller_arguments` event, triggered after controller arguments have been resolved
```

Comment:
#18412 has just been merged, you need to rebase because of the changelog though.